### ISSUE TITLE
Required Field In Module

### ISSUE THREAD

**Issue by umertoday** [elegantthemes/create-divi-extension#485]
### Problem Description

 Is there way to make field required for divi module ?
### Steps To Reproduce
1.

**Comment by umertoday** [comment]
Waiting For answer................................................. ?

**Comment by lanresmith** [comment]
@umertoday sorry for the delay. Do you mean making a field in the settings modal for a custom module required?

**Comment by umertoday** [comment]
yes if i want to know how to make some fied that should be required for custom module or existing module and prompt error tonuset to fill that field

**Comment by lanresmith** [comment]
Sorry, there's no native support for that. You might want to [create a custom field](https://www.elegantthemes.com/documentation/developers/how-to-create-a-custom-field-for-a-divi-builder-module/) instead & use the `_onChange` handler.

**Comment by UmerCheema-WPD** [comment]
I know about it. But it is useless because I want to stop user to save data and give error to fill field must.
![Screenshot_1](https://user-images.githubusercontent.com/69952493/128069427-12df7ad9-e8c0-4edb-9938-53c7295af67e.png)

**Comment by umertoday** [comment]
I know about it. But it is useless because I want to stop user to save data and give error to fill field must.
![Screenshot_1](https://user-images.githubusercontent.com/9529251/128069799-47eec438-e977-4bc3-9ebe-664fa607dd2e.png)
